<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogScanReceipt</class>
 <widget class="QDialog" name="DialogScanReceipt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ФН</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_fn"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>ФД</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_fd"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>ФПД</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_fpd"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonManualInput">
       <property name="text">
        <string>Manual</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="cwImage" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>cwImage</class>
   <extends>QWidget</extends>
   <header>cwimage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
